### PR TITLE
[CC Automation] Update API test to use refresh-all and add CLI test for bulk actions

### PR DESCRIPTION
**Pull request by sambible**
### Problem Statement
Adds test coverage for: https://bugzilla.redhat.com/show_bug.cgi?id=2126420 / https://issues.redhat.com/browse/SAT-18199

### Related Issues
Also updates robottelo CLI to support `hammer alternate-content-source bulk` actions, and the API test to cover the endpoint added in https://github.com/SatelliteQE/nailgun/pull/1086

PRT test Cases example
trigger: test-robottelo
pytest: tests/foreman/api/test_acs.py -k 'test_positive_run_bulk_actions'

trigger: test-robottelo
pytest: tests/foreman/cli/test_acs.py -k 'test_bulk_actions'

### CODE REVIEW

File: tests/foreman/api/test_acs.py [hunk 1]
```diff
@@ -116,6 +116,7 @@ def test_positive_run_bulk_actions(module_target_sat, module_yum_repo):
     :steps:
         1. Create several ACSes.
         2. Bulk refresh them all.
+        3. Add 1 more ACS, then use bulk refresh all.
         3. Bulk destroy some of them.
```
Comment (vsedmik) [inline tests/foreman/api/test_acs.py:120]: This should fix `make docs` in CI/CQ
```suggestion
        4. Bulk destroy some of them.
        5. Cleanup the rest.
```

**Review by vsedmik — CHANGES_REQUESTED** [review]
One minor thing, otherwise looks good to me :+1: 
Let's kick on PRT

**Review by jyejare — APPROVED** [review]
Framework changes look good

**Comment by sambible** [comment]
trigger: test-robottelo
pytest: tests/foreman/api/test_acs.py -k 'test_positive_run_bulk_actions'

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 7535
Build Status: SUCCESS
PRT Comment: pytest tests/foreman/api/test_acs.py -k test_positive_run_bulk_actions --external-logging
Test Result : =========== 1 passed, 6 deselected, 15 warnings in 707.43s (0:11:47) ===========
```

**Comment by sambible** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_acs.py -k 'test_bulk_actions'

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 7541
Build Status: SUCCESS
PRT Comment: pytest tests/foreman/cli/test_acs.py -k test_bulk_actions --external-logging
Test Result : ========== 1 passed, 13 deselected, 22 warnings in 658.79s (0:10:58) ===========
```

**Review by vsedmik — CHANGES_REQUESTED** [review]
Doc strings still need a fix to satisfy CI/CQ.

File: robottelo/cli/acs.py [hunk 2]
```diff
@@ -9,7 +9,7 @@
     [ARG] ...       Subcommand arguments
 
 Subcommands::
-
+    bulk            Modify alternate content sources in bulk
```
Comment (vsedmik) [inline robottelo/cli/acs.py:12]:
```suggestion

    bulk            Modify alternate content sources in bulk
```

**Review by vsedmik — CHANGES_REQUESTED** [review]
Should be the last one missing

**Comment by vsedmik** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_acs.py -k 'test_bulk_actions'

**Review by vsedmik — APPROVED** [review]
All green

**Pull request merged**